src/main/java/frc/robot/RobotContainer.java

```diff
@@ -35,31 +35,32 @@
  * commands, and button mappings) should be declared here.
  */
 public class RobotContainer {
-  private Joystick m_driveJoystick;
-  private Joystick m_operatorJoystick;
-
-  private DriveTrain m_driveTrain;
-  private Shooter m_shooter;
-  private Climber m_climber;
-  private Limelight m_limelight;
-  private PowerDistributionPanel m_pdp;
-  private Revolver m_revolver;
-  private Intake m_intake;
-  private Turret m_turret;
+  private final Joystick m_driveJoystick;
+  private final Joystick m_operatorJoystick;
+  private final DriveTrain m_driveTrain;
+  private final Shooter m_shooter;
+  private final Climber m_climber;
+  private final Limelight m_limelight;
+  private final PowerDistributionPanel m_pdp;
+  private final Revolver m_revolver;
+  private final Intake m_intake;
+  private final Turret m_turret;
 
   /**
    * The container for the robot. Contains subsystems, OI devices, and commands.
    */
   public RobotContainer() {
-    m_limelight = new Limelight();
-    m_pdp = new PowerDistributionPanel(RobotMap.CAN.PDP);
+    m_driveJoystick = new Joystick(RobotMap.JOYSTICK.DRIVE_JOYSTICK);
+    m_operatorJoystick = new Joystick(RobotMap.JOYSTICK.OPERATOR_JOYSTICK);
     m_driveTrain = new DriveTrain();
     m_shooter = new Shooter();
     m_climber = new Climber();
+    m_limelight = new Limelight();
+    m_pdp = new PowerDistributionPanel(RobotMap.CAN.PDP);
     m_revolver = new Revolver(m_pdp);
     m_intake = new Intake();
     m_turret = new Turret();
-
+    
```

Comment:
Nitpick: Trailing spaces again.

Reply:
Do you want me to delete the space in between     m_turret = new Turret();
and configureButtonBindings();

Reply:
Fixed in commit f0ee4bb

Reply:
Actually, I would have liked to keep the newline, just without the trailing spaces. I think it makes organizational sense to group the subsystem initializations together, but separate the button bindings, since they're distinct behaviours.

Would you mind opening another PR for this tiny fix? Just re-add the newline. Thanks :)